UPGRADING.rst

```diff
@@ -11,6 +11,18 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#4190 Create SNS topic for monitoring and security notifications
+================================================================
+
+A new environment variable ``AZUL_NOTIFICATION_EMAIL`` has been added. In
+personal deployments, set this variable to inherit its value from
+``AZUL_OWNER``. As always, use the sandbox deployment's ``environment.py`` as a
+model when upgrading personal deployments.
```

Comment:
Why so complicated? Interpolating one variable into another isn't "inheritance". Just write what string people need to set the variable to. It's a fixed string: `'{AZUL_OWNER}'`.